app/server/lib/DocApi.ts

```diff
@@ -1734,11 +1733,24 @@ export class DocWorkerApi {
     return docAuth.docId!;
   }
 
-  private _getDownloadOptions(req: Request, name: string): DownloadOptions {
+  private async _getDownloadFilename(req: Request, tableId?: string): Promise<string> {
+    const addContext = isAffirmative(process.env.GRIST_INCLUDE_CONTEXT_TO_DOWNLOAD_FILENAMES || false);
```

Comment:
the `|| false` seems not necessary to me, `isAffirmative` is meant to equal false if it is `null` or `undefined`:

https://github.com/gristlabs/grist-core/blob/c4c9a8a5433caa165f359541eadda9430d3cab51/app/common/gutil.ts#L935-L937

Reply:
What do you try exactly in your browser console? I get this:
```js
 function isAffirmative(parameter) { 
   return ['1', 'on', 'true', 'yes'].includes(String(parameter).toLowerCase()); 
 }

isAffirmative(undefined);
// >>> false
```